File: benchmark/2026-09-17-7677581(latest)/semantic-compression/witnesses/kotlin/chunk_B/f10p2_V4_string_left/w.kt
// F10.P2 kotlin WITNESS V4 (left operand kotlin.String: `+` is String.plus(Any?)).
fun probe(i: String, d: Double): String {
    // BEGIN PROBE F10.P2
    val sum = i + d
    // END PROBE F10.P2
    return sum
}
fun main() { val r = probe("3", 0.5); println("sum=" + r + " cls=" + (r as Any).javaClass.name) }

File: benchmark/2026-09-17-7677581(latest)/semantic-compression/probes/witnesses/F10.P2/go/c2/w.go
package main

import "fmt"

func probe() float64 {
	var i float32 = 0.1
	var d float64 = 0
// BEGIN PROBE F10.P2
sum := float64(i) + d
// END PROBE F10.P2
	return sum
}

func main() { fmt.Println(probe()) }
